package.json

```diff
@@ -27,6 +27,7 @@
   },
   "dependencies": {
     "@cashu/cashu-ts": "2.1.0",
+    "@jbojcic/bc-ur": "https://github.com/jbojcic1/bc-ur",
```

Comment:
can you please do `bun remove @jbojcic/bc-ur` and then `bun add https://github.com/jbojcic1/bc-ur` again because I pushed a new version which uses Big.js instead of Bignumber.js to make the size smaller